### DISCUSSION TITLE
Customization of sliderball follow circle!

### DISCUSSION THREAD

**Discussion by aurelia-7** [ppy/osu#27477] [Ideas]
Size adjusting and an option to scale according to CS just like the setting for cursor size! Maybe colour customization too.
![image](https://github.com/ppy/osu/assets/162023183/f7486f97-138e-4684-8132-c30bb5bb99af)
![image](https://github.com/ppy/osu/assets/162023183/205b263d-1b78-409a-b333-76ba4c393c08)

**Comment by respektive** [comment]
I think the follow circle size would be better suited as a setting in the skin editor, but definitely something that would be nice to have.

As for the automatic size based on CS, unless I'm missing something, I believe that it's already doing that by default so that it scales properly with the sliders. Maybe it could be a toggle in the skin editor too, but I don't really see the point.

**Comment by bdach** [comment]
The follow circle size and the slider ball size are *relevant to gameplay mechanics*. I argue that for that reason they should not be modifiable by the user. Except for maybe in a mod.

The only reason weird stable slider ball handling (that entire "actual follow area is not the same size as the _visible_ follow area" thing) continues to work is because people got used to it.

**Reply by Stoppedpuma** [reply]
> Except for maybe in a mod.

It would be a nice option in strict tracking to have customisable follow circle size, [mcosu](https://github.com/McKay42/McOsu) has a mod which is basically strict tracking but the follow area is only the ball which I'm sure plenty of people have used.
